lisp/casual-isearch-utils.el

```diff
@@ -27,7 +27,9 @@
 (require 'casual-lib)
 
 (defconst casual-isearch-unicode-db
-  '((:previous . '("↑" "Previous"))
+  '((:first . '("⏮️" "First"))
+    (:last . '("⏭️" "Last"))
+    (:previous . '("↑" "Previous"))
     (:next . '("↓" "Next")))
```

Comment:
Please modify `test-casual-isearch-unicode-get` to support testing these changes.

https://github.com/kickingvegas/casual/blob/main/tests/test-casual-isearch-utils.el